packages/-ember-data/tests/integration/relationships/one-to-many-test.js

```diff
@@ -126,6 +131,18 @@ module('integration/relationships/one_to_many_test - OneToMany relationships', f
             attributes: {
               title: 'EmberFest was great',
             },
+            relationships: {
+              user: {
+                // data: {
```

Comment:
uncomment these lines, and the test becomes green

Reply:
Thanks! definitely had hoped the current state of the internals would have cleared this one up, but shouldn't be too hard to fix.

Reply:
happy to pair with you on it, can teach you a bit about how it works that way :)

Reply:
👍